### DISCUSSION TITLE
Add native favicons for sidebar WEB panel "tabs"

### DISCUSSION THREAD

**Discussion by paillp** [Floorp-Projects/Floorp#628] [Feature Request]
Currently, icons in the sidebar WEB panels are fetched (I guess) from the icon provider, which doesn't seem to work very well, especially when applications are accessed through redirects / SSO etc...

This is a bit of a problem to me since I intended to use sidebar links as "pinned tabs" and use them as a panel I could toggle (email and other stuff I access regularly).

However, since the icons are fetched from Google / DDG rather than fetching the actual favicon, it renders the sidebar quite ugly and impractical.

See the attached picture.

![image](https://github.com/Floorp-Projects/Floorp/assets/68086605/8fdcd8d0-79d9-41e0-8bd5-2ed7bcb2a778)

It would be great to have the choice to fetch favicons from the remote servers rather than Google / DDG.

Edit:
I know it's been kinda rejected ([issue 613](https://github.com/Floorp-Projects/Floorp/issues/613)), but a work around would be to let users choose an image (typically favicon or SVG) of their own.

**Comment by SystemError12** [comment]
Agree. The sidebar is good and one of the things I like alot in Floorp. Why not improve on it and make it better. The icons is one thing, the other is the width of the sidebar. Could we have the width customizable as well as larger icons?
